### PR TITLE
Fake TCC variable

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

set NWP to cache, so that ocf_datapipes uses it

added in a task to remove it #31 

## How Has This Been Tested?

CI tests

- [ ] Yes

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
Selecting Channels: 
('hcc', 'lcc', 'mcc', 'prate', 'sde', 'sr', 't2m', 'tcc', 'u10', 'u100', 'u200', 'v10', 'v100', 'v200') 
out of 

['clt' 'dlwrf' 'dswrf' 'duvrs' 'hcc' 'lcc' 'mcc' 'prate' 'sde' 'sr' 't2m', 'u10' 'u100' 'u200' 'v10' 'v100' 'v200' 'vis']

So, we are missing `tcc`

Need to fake this for the moment

File: india_forecast_app/models/pvnet/model.py [hunk 1]
```diff
@@ -132,6 +132,9 @@ def _prepare_data_sources(self):
         # Process/cache remote zarr locally
         process_and_cache_nwp(nwp_source_file_path, nwp_path)
 
+        # set NWP_ZARR_PATH as this new path
+        os.environ["NWP_ZARR_PATH"] = nwp_path
```
Comment (confusedmatrix) [inline india_forecast_app/models/pvnet/model.py:136]: OCF-datapipes should be reading the zarr from the config on L162, rather than from the env var

Comment (peterdudfield) [reply india_forecast_app/models/pvnet/model.py:136]: you got a link for that?

Comment (peterdudfield) [reply india_forecast_app/models/pvnet/model.py:136]: you are right, let me remove that

**Comment by confusedmatrix** [comment]
All LGTM now!

**Pull request merged**